dev/Tiltfile

```diff
@@ -198,6 +223,11 @@ for namespace in pg_namespaces:
   k8s_yaml(namespace_inject(pg_operator_bundle_yaml, namespace))
   k8s_resource(
     workload='percona-postgresql-operator%s' % (':deployment:' + namespace if len(pg_namespaces) > 1 else ''),
+    objects=[
+      'percona-postgresql-operator:serviceaccount:%s' % namespace,
+      'percona-postgresql-operator:role:%s' % namespace,
+      'percona-postgresql-operator:rolebinding:%s' % namespace,
```

Comment:
Thanks for adding this back, this was the resource that broke when I upgraded to PGO 2.5.0 and I just [removed it to quickly fix the error](https://github.com/percona/everest/commit/bb881cbbf006c1ddd87a7962ad9dad3f3487e2d3)